library/core/src/clone.rs

```diff
@@ -157,6 +157,8 @@ mod uninit;
 #[lang = "clone"]
 #[rustc_diagnostic_item = "Clone"]
 #[rustc_trivial_field_reads]
+#[rustc_const_unstable(feature = "const_clone", issue = "142757")]
```

Comment:
I dont think we need per trait gates? Seems like it could cause ppl to add lots of feature gates. And we can stabilize individual const traits without stabilizing the full set either way. Library feature gates are funny this way